src/EditorFeatures/Text/Extensions.SnapshotSourceText.cs

```diff
@@ -88,7 +89,27 @@ private static SnapshotSourceText CreateText(ITextSnapshot editorSnapshot)
                 var text = strongBox.Value;
                 if (text != null && text._reiteratedVersion == editorSnapshot.Version.ReiteratedVersionNumber)
```

Comment:
Have we done any history digging to understand if this is of any actual value? I agree that it's In Theory™ a safe optimization, but I do have to ask if it's worth it. :-/